### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by 2TomLi**
**Summary**
update the contributing.md to add VSIX instructions.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -93,3 +93,7 @@ Follow these steps to update a locale for the SAS Extension for VSCode:
 | **German**               | de                 | David Weik  |
 | **Chinese (Simplified)** | zh-cn              | Wei Wu      |
 | **Portuguese (Brazil)**  | pt-br              | Mark Jordan |
+
+### Download packaged VSIX files
```
Comment (scottdover) [inline CONTRIBUTING.md:97]:
```suggestion
## Download packaged VSIX files

```

Comment (scottdover) [reply CONTRIBUTING.md:97]: This should likely be an h2 to better align with the rest of the doc

**Comment by scnwwu** [comment]
Covered in #701

**Comment by scottdover** [comment]
Closing this one as it was handled in another PR. Feel free to re-open as needed :)

**Pull request closed without merging**